### PR TITLE
metrics: add owner in memory table count metrics

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

part 2 of https://github.com/pingcap/ticdc/issues/1533

### What is changed and how it works?

Add table count in metric, with labels of `changefeed-id`, `capture-addr` and `type`

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

- No release note

### CODE REVIEW

File: cdc/metrics_owner.go [hunk 1]
```diff
@@ -37,11 +37,24 @@ var (
...
+			Namespace: "ticdc",
+			Subsystem: "owner",
+			Name:      "maintain_table_num",
+			Help:      "number of replicated tables maintained in owner",
+		}, []string{"changefeed", "capture", "type"})
+)
+
+const (
+	maintainTableTypeTotal string = "total"
+	maintainTableTypeWip   string = "wip"
```
Comment (overvenus) [inline cdc/metrics_owner.go:51]: Could you add some comments about these two types?

**Review by overvenus — COMMENTED** [review]
rest LGTM

**Review by leoppro — APPROVED** [review]

**Comment by liuzix** [comment]
LGTM

**Comment by amyangfei** [comment]
> LGTM

need `/lgtm` @liuzix ,thx

**Pull request merged**